docs/docs/actions.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Actions
-description: Documentation on actions and how they help you manipulate state within Gatsby
+title: एक्शन्स
+description: एक्शन्स पर प्रलेखन और वे आपको Gatsby के भीतर स्टेट में हेरफेर करने में कैसे मदद करते हैं
 jsdoc:
   - "gatsby/src/redux/actions/public.js"
   - "gatsby/src/redux/actions/restricted.js"
 contentsHeading: Functions
 ---
 
-Gatsby uses [Redux](http://redux.js.org) internally to manage state. When you implement a Gatsby API, you are passed a collection of actions (equivalent to actions bound with [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux) which you can use to manipulate state on your site.
+Gatsby स्टेट का प्रबंधन करने के लिए आंतरिक रूप से [Redux](http://redux.js.org) का उपयोग करता है। जब आप एक Gatsby API कार्यान्वित करते हैं, तो आपको क्रियाओं का एक संग्रह  (equivalent to actions bound with [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux) जिसका उपयोग आप स्टेट में हेरफेर करने के लिए कर सकते हैं आपकी साइट पर ।
```

Comment:
`(equivalent to actions bound with [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux)` => `(Redux में [bindActionCreators](https://redux.js.org/api/bindactioncreators/) से बंधे हुए एक्शन्स के बराबर)`